src/app/util/ember-compatibility-functions.cpp

```diff
@@ -461,5 +461,21 @@ CHIP_ERROR WriteSingleClusterData(ClusterInfo & aClusterInfo, TLV::TLVReader & a
                                                   Protocols::SecureChannel::Id, imCode);
 }
 
+void emberAfReportingAttributeChangeCallback(EndpointId endpoint, ClusterId clusterId, AttributeId attributeId, uint8_t mask,
```

Comment:
This function should be placed in global namespace.

Reply:
I just remove this function, i prefer you keep. this function in your pr, since i don't have test in my pr for this part